docs/release-notes/2.41.1.md

```diff
@@ -0,0 +1,42 @@
+
+| **Release date** | Jan 24, 2024                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations—regardless of whether your databases are located on-premises or in the cloud.
+
+## Release Highlights
+
+### Improved support for Arbiter nodes
+
+We've made it easier to add and monitor MongoDB Arbiter nodes in PMM, as this no longer requires workarounds for cases where authorization is enabled on the MongoDB replica set. Additionally, Arbiter nodes are now displayed correctly on the dashboards:
+
+![!](../_images/Arbiter.png)
+
+### New metric for sharded MongoDB insights
+
+We've added a new metric to our MongoDB exporters: `mongodb_shards_collection_chunks_count`. This metric is available for sharded Mongo instances, offering insights into the distribution of collection chunks across different shards.
+
+While this version is not currently integrated into any dashboard, you have the flexibility to incorporate it into your custom dashboards. For comprehensive instructions on managing dashboards, check out [this video tutorial](https://www.youtube.com/watch?v=vk1QBiMVzz4) and the [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-manage-dashboards.html).
+
+### New experimental MongoDB Instance Summary dashboard
+
+This release also introduces a new MongoDB dashboard that offers a simplified view of the most critical MongoDB metrics. You can find the new **MongoDB Instance Summary** dashboard in the **Experimental** folder of your PMM installation. We encourage you to evaluate its performance with your MongoDB instance and share your valuable feedback on our forum.
+
+## Improvements
+
+- [PMM-12390](https://jira.percona.com/browse/PMM-12390) - [Backup Management]: The **Service name** field on the Create Backup pages is now case-insensitive, allowing for more comprehensive and accurate results retrieval.
+- [PMM-12712](https://jira.percona.com/browse/PMM-12712) - MongoDB sharded cluster metrics now include a new metric to represent database and collection distribution across different shards. This helps gain insights into the overall balance of shards within MongoDB setups.
+- [PMM-12510](https://jira.percona.com/browse/PMM-12510) - Improved workflow for adding MongoDB Arbiter nodes to PMM, which also ensures that Arbiter nodes are now displayed correctly on the dashboards.
+
+## Bugs Fixed
+
```

Comment:
- [PMM-9407] (https://perconadev.atlassian.net/browse/PMM-9407): Fixed issue with executing Custom Queries for PostgreSQL on Debian due to the absence of queries.yaml within packages.
- [PMM-12290](https://perconadev.atlassian.net/browse/PMM-12290) [QAN] Fixed bugs with SQL comments parsed in queries when they were not intentionally enabled for remotely added PostgreSQL and MySQL.
- [PMM-12350](https://perconadev.atlassian.net/browse/PMM-12350) Fixed problem with mongod_exporter flood logs when connected to Mongos. 
- [PMM-12738](https://perconadev.atlassian.net/browse/PMM-12738) Fixed bug with inability to run PMM using Helm with customer certificates.  
- [PMM-12781](https://perconadev.atlassian.net/browse/PMM-12781) Fixed a permission error that flooded PostgreSQL logs by fixing the pg_ls_waldir function's execution permissions.